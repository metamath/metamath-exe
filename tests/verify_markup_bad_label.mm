$c c $. $v v $.
vv $f c v $.
com1 $a c v $.
mm $a c v $.
$( $t
  latexdef 'c' as 'c';
  htmldef 'c' as 'c';
  althtmldef 'c' as 'c';
  latexdef 'v' as 'v';
  htmldef 'v' as 'v';
  althtmldef 'v' as 'v';
$)
