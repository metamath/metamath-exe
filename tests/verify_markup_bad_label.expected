MM> READ "verify_markup_bad_label.mm"
Reading source file "verify_markup_bad_label.mm"... 206 bytes
206 bytes were read into the source buffer.
The source has 5 statements; 2 are $a and 0 are $p.
No errors were found.  However, proofs were not checked.  Type VERIFY PROOF *
if you want to check them.
MM> Continuous scrolling is now in effect.
MM> Checking statement label conventions...
?Warning: In statement "com1" at line 3 in file "verify_markup_bad_label.mm". 
    To workaround a Microsoft operating system limitation, the the following
    reserved words cannot be used for label names: CON, PRN, AUX, CLOCK$, NUL,
    COM1, COM2, COM3, COM4, COM5, COM6, COM7, COM8, COM9, LPT1, LPT2, LPT3,
    LPT4, LPT5, LPT6, LPT7, LPT8, and LPT9.  Also, "mm*.html" is reserved for
    Metamath file names.  Use another name for this label.
?Warning: In statement "mm" at line 4 in file "verify_markup_bad_label.mm".  To
    workaround a Microsoft operating system limitation, the the following
    reserved words cannot be used for label names: CON, PRN, AUX, CLOCK$, NUL,
    COM1, COM2, COM3, COM4, COM5, COM6, COM7, COM8, COM9, LPT1, LPT2, LPT3,
    LPT4, LPT5, LPT6, LPT7, LPT8, and LPT9.  Also, "mm*.html" is reserved for
    Metamath file names.  Use another name for this label.
Checking latexdef, htmldef, althtmldef...
Checking statement comments...
Checking section header comments...
Checking bibliographic references...
Checking mathbox independence...
